### PR TITLE
[*] BO : Pass product object on displayAdminProductsExtra hook

### PR DESCRIPTION
**Pull request by soullivaneuh**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | This an override application proposed [here](https://www.prestashop.com/forums/topic/221136-solved-new-tab-in-product-back-office/?p=1090614). Replaces #5095. |
| Type? | improvement |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | N/A |
| How to test? | N/A |

### CODE REVIEW

**Comment by julienbourdeau** [comment]
How about calling it 'product' instead of 'obj' ?

**Comment by Soullivaneuh** [comment]
@julienbourdeau like this? :-)

File: controllers/admin/AdminProductsController.php [hunk 1]
```diff
@@ -2597,7 +2597,7 @@ public function renderKpis()
         $helper->refresh = (bool)(ConfigurationKPI::get('8020_SALES_CATALOG_EXPIRE') < $time);
         $moduleManagerBuilder = new ModuleManagerBuilder();
         $moduleManager = $moduleManagerBuilder->build();
-    
+
```
Comment (Shine-neko) [inline controllers/admin/AdminProductsController.php]: ?

**Comment by julienbourdeau** [comment]
Sorry, I just realized you're modifying "AdminProductsController", this controller is no longer used in PrestaShop 1.7. The product page have been redone with symfony. Now you get the id_product in this hook
https://github.com/PrestaShop/PrestaShop/blob/674c025501516390f60db2dc0d27c6b1313f3fab/src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig#L939-L948

File: controllers/admin/AdminProductsController.php (hunk 1 above)
Comment (Soullivaneuh) [inline controllers/admin/AdminProductsController.php]: Trailling whitespaces auto corrected by my IDE.

BTW, trailling whitespaces is bad. :smile:

Comment (julienbourdeau) [inline controllers/admin/AdminProductsController.php]: 👍

**Pull request closed without merging**